README.md

```diff
@@ -1,29 +1,122 @@
-# RdTools
+# About RdTools
 
-Degradation Tools
+RdTools is a set of Python tools for analysis of photovoltaic data.
+In particular, PV production data is evaluated over several years
+to obtain rates of performance degradation over time. Rdtools can
+handle both high frequency (hourly or better) or low frequency (daily, weekly, etc.)
+datasets. Best results are obtained with higher frequency data.
 
-Python tools for photovoltaic degradation analysis.
+Full examples are worked out in the example notebooks in [rdtools/docs](./docs/degradation_example.ipynb).
 
-## Install using pip
+
+## Workflow
+
+0. Import and preliminary calculations
+1. Normalize data using a performance metric
+2. Filter data that creates bias
+3. Aggregate data
+4. Analyze aggregated data to estimate the degradation rate
+
+
+<img src="./screenshots/Workflow1.png" width="600" height="331" alt="RdTools Workflow"/>
+
+## Degradation Results
+
+RdTools output shows filtered historical data along with a linear trend line.
+However, the most accurate estimate of degradation over the analysis period in question
+is the Year-on-year (YOY) distribution, which is a normal distribution, the central tendancy
+of which is the most representative degradation rate.  Distribution width provides information
+about the accuracy of the analysis.
+
+<img src="./screenshots/Clearsky_result.png" width="600" height="456" alt="RdTools Result"/>
+
+
+Two workflows are available for system performance ratio calculation, and illustrated in an example notebook. 
+The sensor-based approach assumes that site irradiance and temperature sensors are calibrated and in good repair.
+Since this is not always the case, a 'clear-sky' workflow is provided that is based on
+modeled temperature and irradiance.  Note that site irradiance data is still required to identify
+clear-sky conditions to be analyzed.  In many cases, the 'clear-sky' analysis can identify conditions
+of instrument errors or irradiance sensor drift, such as in the above analysis.
+
+
+## Install RdTools using pip
+
+RdTools can be installed into Python from the command line.
 
 1. Clone or download the rdtools repository.
-2. Navigate to repository: `cd rdtools`
+2. Navigate to the repository: `cd rdtools`
 3. Install via pip: `pip install .`
```

Comment:
Just a reminder: the install instructions will need to be changed to `pip install rdtools` once released to PyPI. Steps 1 and 2 will not be needed.